translations/hu.json

```diff
@@ -0,0 +1,59 @@
+{
+  "howItWorks": "Hogyan vásárolj Bitcoint KYC nélkül",
```

Comment:
If you look at the homepage preview, the peach icon + language icon aren't visible